### PR TITLE
Defect/enrolled grade used instead of asmt grade

### PR DESCRIPTION
**Pull request by esotrope**

### CODE REVIEW

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 1]
```diff
@@ -125,7 +125,7 @@
       },
       "11": {
         "long-name": "Grade 11",
-        "short-name": "G11"
+        "short-name": "High School"
```
Comment (esotrope) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: Do we want G9-12 to be "High School" ? or just G11?

Comment (malaffoon) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: i think the short form is `HS`?

Comment (esotrope) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: Yeah I'm looking at the UI and realized i need to be setting the long-name as well.
Same Q stands though - do we want G9-12 to say HS?

Comment (jtreuting) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: Yeah i think short name should be `HS`

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 2]
```diff
@@ -124,8 +124,8 @@
         "short-name": "G10"
       },
       "11": {
-        "long-name": "Grade 11",
-        "short-name": "G11"
+        "long-name": "High School",
+        "short-name": "HS"
```
Comment (esotrope) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: changed grade 11 text to HS/High School

Comment (esotrope) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: changed grade 11 text to HS/High School

**Pull request merged**